templates/init-migrate-hook.yaml

```diff
@@ -53,16 +37,46 @@ spec:
       labels:
         {{- include "quorumkeymanager.labels" . | nindent 8 }}
     spec:
+      securityContext:
+        fsGroup: 1001
       restartPolicy: Never
       {{- if .Values.imageCredentials }}
       imagePullSecrets:
         - name: {{ include "quorumkeymanager.imagePullSecretName" . }}
       {{- end }}
+      {{- if .Values.postgresql.tls.enabled }}
+      initContainers:
+        - name: init-chmod-data
+          image: docker.io/bitnami/bitnami-shell:10-debian-10-r185
+          imagePullPolicy: "Always"
+          resources:
+            requests:
```

Comment:
why you need to limit the resources?

Reply:
Well this was taken from an exemple, as is, that ensures the Pod to be properly scheduled by k8s